sympy/physics/mechanics/tests/test_functions.py

```diff
@@ -165,6 +166,12 @@ def test_find_dynamicsymbols():
     sol = {x, y.diff(), y, x.diff().diff(), z, z.diff()}
     assert find_dynamicsymbols(expr) == sol
     # Test finding all but those in sym_list
-    exclude = [x, y, z]
+    exclude_list = [x, y, z]
     sol = {y.diff(), x.diff().diff(), z.diff()}
-    assert find_dynamicsymbols(expr, exclude) == sol
+    assert find_dynamicsymbols(expr, exclude=exclude_list) == sol
```

Comment:
It is best not to change the API if at all possible. Why is this change required?

Reply:
ignore this comment

Reply:
@moorepants I have made a new commit addressing the requested changes. Please review.
Thanks.